pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.output

```diff
@@ -18,7 +18,7 @@ build0-gstfj-ci-tests-worker-b-jv5bg      Degraded      Draining   4.16.0-ec.2
 build0-gstfj-ci-tests-worker-b-kj6gk      Degraded      Draining   4.16.0-ec.2   ?     failed to drain node: <node> after 1 hour. Please see machine-config-controller logs for more information
 build0-gstfj-ci-tests-worker-c-dcz9p      Degraded      Draining   4.16.0-ec.2   ?     failed to drain node: <node> after 1 hour. Please see machine-config-controller logs for more information
 build0-gstfj-ci-prowjobs-worker-d-ddnxd   Unavailable   Pending    ?             ?     Machine Config Daemon is processing the node
-build0-gstfj-ci-tests-worker-b-d9vz2      Unavailable   Pending    ?             ?     node is reporting NotReady=False with message="kubelet is still setting up stuff"
+build0-gstfj-ci-tests-worker-b-d9vz2      Unavailable   Pending    ?             ?     Node is not ready
```

Comment:
I'm wondering if `Not ready` / `Disk pressure` / `Network unavailable` could be enough - "Node is" is kinda implied by the message being shown in the node table.

Reply:
Done